modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java

```diff
@@ -101,6 +131,7 @@ public void render(
 		ContentDashboardAdminDisplayContext
 			contentDashboardAdminDisplayContext =
 				new ContentDashboardAdminDisplayContext(
+					assetVocabularyMetricOptional.orElse(null),
```

Comment:
Why would we use an optional,  if we handle it as null when it is empty?

Reply:
I don't want to pass an Optional as parameter. I first though creating an empty AssetVocabularyMetric, would it be a better approach?

Reply:
I would love that solution, but if you are going to handle it as null, return null is Ok too.